<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>thongtin</class>
 <widget class="QMainWindow" name="thongtin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>254</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>CHỈNH SỬA THÔNG TIN CÁ NHÂN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnHuy">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>180</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Huỷ</string>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/res/res/icon/discard.png</normaloff>:/res/res/icon/discard.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLuu">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>180</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Lưu</string>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/res/res/icon/save.png</normaloff>:/res/res/icon/save.png</iconset>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>131</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>121</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="res.qrc">:/res/res/icon/personal-card.png</pixmap>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="leditAdress">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>140</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="leditName">
    <property name="geometry">
     <rect>
      <x>271</x>
      <y>80</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>60</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>80</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tên khách hàng</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leditPhone">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>110</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>12</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>159</x>
      <y>110</y>
      <width>80</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>SĐT                    </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>140</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Địa chỉ             </string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>490</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
